apps/voting/contracts/test/mocks/VotingMock.sol

```diff
@@ -4,6 +4,11 @@ import "../../Voting.sol";
 
 
 contract VotingMock is Voting {
+    uint64 mockTime = uint64(now);
```

Comment:
When using this contract behind a proxy, this value will not be set and mockTime will be 0 until mock_setTimestamp is used.

Reply:
Turns out the tests are calling `VotingMock.mock_setTimestamp()` on creation of the proxy so this wasn't a problem, but I can't figure out a better way of doing this outside of making it good practice to set this immediately on mocks.

I was thinking about adding an `initialize()` wrapper that would set this, but it doesn't feel right.